recipes/torchcodec/recipe.yaml

```diff
@@ -0,0 +1,135 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+
+context:
+  name: torchcodec
+  version: "0.2.1"
+  build_number: 0
+  torch_proc_type: ${{ "cuda" ~ cuda_compiler_version | version_to_buildstring if cuda_compiler_version != "None" else "cpu" }}
+  string_prefix: ${{ cuda_build_string if cuda ==  "true" else "cpu_" }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/pytorch/torchcodec/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: b142ef4fef1e9ddb6dde70dda6ac4d12a02a010065230468eadbf37478621927
+
+build:
+  number: ${{ build_number }}
+  skip: 
+    # temporarily, we are running out of space in staged-recipes and will fix this in the feedstock
+    - cuda_compiler_version != "None"
+    # PyTorch does not support cuda 11
+    - cuda_compiler_version == "11.8"
+    # Upstream does not support Windows at the moment https://github.com/pytorch/torchcodec/issues/522
+    - win
+  string: ${{ torch_proc_type }}_py${{ python | version_to_buildstring }}_h${{ hash }}_${{ build_number }}
+  script:
+    file: build
```

Comment:
I think that's redundant.

Reply:
Indeed it is. I added it as my personal preference is to avoid "implicit" behaviors as the person that reads the recipe needs to be aware of this "implicit" behaviors to understand what is going on (I was personally quite confused by the change from `bld.bat` to `build.bat` in the implicit called script from recipe v0 to v1, but I do not have any strong feeling on this, so if the consensus is to remove it, for me it is fine to remove it.